docs/product-migration-guide/src/main/asciidoc/migration-configure-kie-server-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='migration-configure-kie-server-proc']
+= Configure and start {KIE_SERVER}
+
+After you have migrated all {CENTRAL} data and have upgraded the relevant database, configure and start the new {KIE_SERVER} to transfer the execution server functionality and apply the migrated changes.
+
+.Prerequisites
+* Project data from {PRODUCT_OLD} 6.4 has been migrated using the {CENTRAL} migration tool.
+* The {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} has been upgraded.
+* All API references to {CENTRAL} execution server in application code have been redirected to the {KIE_SERVER}.
+
+.Procedure
+Navigate to the {PRODUCT} {PRODUCT_VERSION} `bin` directory and start the new {KIE_SERVER} with the following properties. Adjust the specific properties according to your environment.
+
+[source]
+----
+$ ~/EAP_HOME/bin/standalone.sh - c server-config=standalone-full.xml <1>
+-Dorg.kie.server.id=myserver <2>
+-Dorg.kie.server.controller=http://localhost:8080/business-central/rest/controller <3>
+-Djboss.socket.binding.port-offset=150 <4>
+-Dorg.kie.server.location=http://localhost:8230/kie-server/services/rest/server <5>
+-Dorg.kie.server.persistence.dialect=org.hibernate.dialect.PostgreSQLDialect <6>
+-Dorg.kie.server.persistence.ds=java:jboss/datasources/psjbpmDS <7>
+----
+<1> Start command with `standalone-full.xml` server profile
+<2> Server ID that must match the server template name defined in {CENTRAL}
+<3> Controller location, {CENTRAL} URL with `/rest/controller` suffix
+<4> Port offset for {KIE_SERVER} location, to avoid port conflicts (not needed if {KIE_SERVER} already uses a separate port)
+<5> {KIE_SERVER} location (including defined port offset, if applicable)
+<6> Hibernate dialect to be used
+<7> JNDI name of the data source used for your previous {PRODUCT_OLD} database
+
+*<@Maciej (2 of 2): Are #4 and #5 in the above properties correct now? Regarding {KIE_SERVER} location. These properties all correct overall now?>*
+
```

Comment:
I want to clarify it a little bit. <5> does not have port offset, it has whole port number. Port offset is only <4> which is only applicable if Business Central and Process Server each run on a different EAP instance. In that case to avoid duplicate port binding, a user has to start one of them either with a port offset or change configuration of the EAP to run on a different port. What scenario do we consider here? Process Server + BC on one EAP or separate instances? Maybe we can at least mention that there are 2 possibilities or is it mentioned elsewhere in documentation? Wdyt @mswiderski ?

Reply:
@MarianMacik, thanks for the clarification. That's what I'm trying to convey. For <4>, I say "Port offset for {KIE_SERVER} location, to avoid port conflicts **(not needed if {KIE_SERVER} already uses a separate port)**" The bold part is to specify what you're saying, as far as I understand it. I'm trying to keep it simple and not get into the various options, but if what I'm saying isn't working or isn't right, I can reconsider.

Also, in <5>, when I say "(including defined port offset, if applicable)", to me that meas what you're saying. That is, the server location includes the offset of 150, meaning 8080 plus 150 is the 8230 shown there. Would it be clearer if I say "(plus the defined port offset value, if applicable)"?

And Marian, are you on any IRC channels that I can ping you on? May be easier that way for some back-and-forth.

Reply:
Sorry, @sterobin, I tried to look at it differently and yes, it makes sense :) Sorry for confusion. And regarding IRC yes, i am regularly on jbpm-dev for example, Maciej is also there.

Reply:
this command line to start kie server is valid if:
- kie server runs on another server instance than business central
- business central runs on default port offset (0) and thus will be available on 8080

so that implicitly says we have environment with two application servers - one hosting business central another one kie server. If that's what we want then it is valid.

Though I had impression that before we had everything in single instance of application server and thus everything running on 8080. In that case it should be like this:

```
+$ ~/EAP_HOME/bin/standalone.sh - c server-config=standalone-full.xml <1>
 +-Dorg.kie.server.id=myserver <2>
 +-Dorg.kie.server.controller=http://localhost:8080/business-central/rest/controller <3>
 +-Dorg.kie.server.location=http://localhost:8080/kie-server/services/rest/server <5>
 +-Dorg.kie.server.persistence.dialect=org.hibernate.dialect.PostgreSQLDialect <6>
 +-Dorg.kie.server.persistence.ds=java:jboss/datasources/psjbpmDS <7>
```

Reply:
@mswiderski and @MarianMacik, last try. This section good now?

Reply:
I have just realized that for server config it is either ```--server-config=standalone-full.xml``` or short version ```-c standalone-full.xml``` I will read it once again tomorrow though.

Reply:
@sterobin Not sure if you saw my comment above :)